.obs/chartfile/operator/templates/channels.yaml

```diff
@@ -28,3 +28,51 @@ spec:
   type: custom
   {{ end }}
 {{ end }}
+---
+{{ $enabled := false }}
```

Comment:
A bit convoluted, but I couldn't find a more concise syntax due to the lookup result being nil if the channel does not exist yet. 

Something like `{{ (lookup "elemental.cattle.io/v1beta1" "ManagedOSVersionChannel" "fleet-default" "sle-micro-5.5-kvm").spec.enabled | default false  }}` would have been great, but it will fail in the nil case. :(